### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by rogertinsley**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/rogertinsley/ef-change-notify/pull/1" target="_blank">ReviewNinja

File: README.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
 ef-change-notify
 ================
 
-A change notification framework for Entity Framework 5.
\ No newline at end of file
+A change notification framework for Entity Framework 5.
+
```
Comment (rogertinsley) [inline README.md:5]: Does this need to be here?

File: README.md [hunk 2]
```diff
@@ -1,4 +1,6 @@
 ef-change-notify
 ================
 
-A change notification framework for Entity Framework 5.
\ No newline at end of file
+A change notification framework for Entity Framework 5.
```
Comment (rogertinsley) [inline README.md:4]: !fix remove this

Comment (rogertinsley) [inline README.md:4]: !fixed it so i did

File: README.md (hunk 1 above)
Comment (rogertinsley) [inline README.md:5]: !resolved yes it does

**Pull request merged**